backend/src/test/kotlin/org/loculus/backend/dbmigration/V1_18_1__Migrate_Sequence_Entries_To_Use_Compression_DictTest.kt

```diff
@@ -0,0 +1,181 @@
+package org.loculus.backend.dbmigration
+
+import com.fasterxml.jackson.databind.node.TextNode
+import mu.KotlinLogging
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.hasSize
+import org.hamcrest.Matchers.`is`
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.loculus.backend.api.GeneticSequence
+import org.loculus.backend.api.ProcessedData
+import org.loculus.backend.controller.OTHER_ORGANISM
+import org.loculus.backend.controller.PublicJwtKeyConfig
+import org.loculus.backend.controller.SPRING_DATASOURCE_PASSWORD
+import org.loculus.backend.controller.SPRING_DATASOURCE_URL
+import org.loculus.backend.controller.SPRING_DATASOURCE_USERNAME
+import org.loculus.backend.controller.datauseterms.DataUseTermsControllerClient
+import org.loculus.backend.controller.files.FilesClient
+import org.loculus.backend.controller.groupmanagement.GroupManagementControllerClient
+import org.loculus.backend.controller.seqsetcitations.SeqSetCitationsControllerClient
+import org.loculus.backend.controller.submission.SOME_LONG_GENE
+import org.loculus.backend.controller.submission.SOME_SHORT_GENE
+import org.loculus.backend.controller.submission.SubmissionControllerClient
+import org.loculus.backend.controller.submission.SubmissionConvenienceClient
+import org.loculus.backend.testutil.TestEnvironment
+import org.loculus.backend.testutil.TestResource
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.DynamicPropertyRegistry
+import org.springframework.test.context.DynamicPropertySource
+
+private val log = KotlinLogging.logger { }
+
+/**
+ * Loads a dump from before this migration into the DB, runs the migration, and tests that sequence entries
```

Comment:
Sorry I am a bit confused isn't this test actually for the second migration V1_18_1 in https://github.com/loculus-project/loculus/pull/5314?